advanced/index.php

```diff
@@ -58,7 +58,7 @@ function setHeader($type = "x") {
 if ($serverName === "pi.hole") {
     // Redirect to Web Interface
     exit(header("Location: /admin"));
-} elseif (filter_var($serverName, FILTER_VALIDATE_IP) || in_array($serverName, $authorizedHosts)) {
+} elseif (filter_var($serverName) || in_array($serverName, $authorizedHosts)) {
```

Comment:
You should not remove `FILTER_VALIDATE_IP` but leave `filter_var()` in there. From the PHP documentation:

> The [second argument is the] ID of the filter to apply. The Types of filters manual page lists the available filters.
> If omitted, `FILTER_DEFAULT` will be used, which is equivalent to `FILTER_UNSAFE_RAW`. This will result in no filtering taking place by default.

This behavior is misleading

Reply:
If `FILTER_VALIDATE_IP` is present then the "website blocked - not found on any Blacklist" splash shows up when the page is accessed from outside the network.

Reply:
Have you tried simply switching the order of the conditionals so it checks the authorized hosts first?

Reply:
I added the `FILTER_SANITIZE_URL` and `FILTER_VALIDATE_URL` and that fixes it too.